src/XMLSecEnc.php

```diff
@@ -59,10 +59,19 @@ class XMLSecEnc
     const URI = 3;
     const XMLENCNS = 'http://www.w3.org/2001/04/xmlenc#';
 
+    /** @var null|DOMDocument */
     private $encdoc = null;
+
+    /** @var null|DOMNode  */
     private $rawNode = null;
+
+    /** @var string */
     public $type = null;
+
+    /** @var DOMElement */
```

Comment:
I will try to complete the missing types - but give me a chance and a little help, I had not implemented it ;)